symfony/phpunit-bridge/3.3/phpunit.xml.dist

```diff
@@ -11,6 +11,8 @@
         <ini name="error_reporting" value="-1" />
         <server name="APP_ENV" value="test" force="true" />
         <server name="SHELL_VERBOSITY" value="-1" />
+        <server name="SYMFONY_PHPUNIT_VERSION" value="6.5" />
```

Comment:
@fabpot @nicolas-grekas I am not really sure if we should use `server` or `env` node here.